builtin/types/transfer_gateway/transfer_gateway.proto

```diff
@@ -546,4 +552,61 @@ message TransferGatewayUpdateBinanceTransferFeeRequest{
 message TransferGatewaySwitchMainnetGatewayRequest {
   Address mainnet_address = 1;
   string gateway_name = 2;
+}
+
+// User hot-wallet
+
+message TransferGatewayAllowedTokenRequest {
+  Address foreign_contract_address = 1;
+  bool is_enabled = 2;
+}
+
+message TransferGatewayListAllowedTokenRequest {
+}
+
+message TransferGatewayListAllowedTokenResponse {
+  repeated TransferGatewayHotWalletAllowedToken allowed_tokens = 1;
+}
+
+message TransferGatewayHotWalletAllowedToken {
+  uint64 index = 1;
+  Address foreign_contract_address = 2;
+  bool is_enabled = 3;
+}
+
+message TransferGatewayHotWalletSummary {
+  Address foreign_contract_address = 1;
+  Address wallet_address = 2;
+}
+
+message TransferGatewayHotWalletAccount {
+  Address owner = 1;
+  uint64 account_index = 2;
+  // mapping between contract index and TransferGatewayHotWalletSummary
+  map<uint64, TransferGatewayHotWalletSummary> wallets = 3;
+}
+
+message TransferGatewayHotWalletRequest {
```

Comment:
Maybe rename `TransferGatewayHotWalletRequest ` ==> `TransferGatewayUserHotWalletGetWalletRequest`. 

`TransferGatewayHotWalletRequest` is a bit vague.